### PR TITLE
C++ Model API support for retinaface resnet50 (to 2021.4.1 release), CVS-61892

### PR DESCRIPTION
**Pull request by fzhar**
Merging separately to release branch

### CODE REVIEW

File: demos/object_detection_demo/cpp/models.lst [hunk 1]
```diff
@@ -19,6 +19,7 @@ person-vehicle-bike-detection-????
 product-detection-0001
 rfcn-resnet101-coco-tf
 retinanet-tf
+retinaface-resnet50-pytorch
```
Comment (eizamaliev) [inline demos/object_detection_demo/cpp/models.lst:22]: Add the 
```
For -at=retinaface-pytorch
```
here.
Also add it to develop branch

**Review by eizamaliev — CHANGES_REQUESTED** [review]

**Comment by eizamaliev** [comment]
cc @vladimir-dudnik

**Comment by fzhar** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**